examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -169,26 +168,34 @@ def score_estimator(estimator, df_test):
     mask = y_pred > 0
     if (~mask).any():
         warnings.warn("Estimator yields non-positive predictions for {} "
-                      "samples out of {}. These will be ignored while "
-                      "computing the Poisson deviance"
+                      "samples out of {}. These predictions will be clipped "
+                      " while computing the Poisson deviance"
                       .format((~mask).sum(), mask.shape[0]))
```

Comment:
Since this is an example, and not in the library, should we print instead of raising a warning? (so that our examples don't raise warnings)